doc/classes/SplitContainer.xml

```diff
@@ -1,22 +1,23 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="SplitContainer" inherits="Container" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
 	<brief_description>
-		A container that splits two child controls horizontally or vertically and provides a grabber for adjusting the split ratio.
+		A container that arranges child controls horizontally or vertically and provides grabbers for adjusting the split ratios between them.
 	</brief_description>
 	<description>
-		A container that accepts only two child controls, then arranges them horizontally or vertically and creates a divisor between them. The divisor can be dragged around to change the size relation between the child controls.
+		A container that arranges child controls horizontally or vertically and creates grabbers between them. The grabbers can be dragged around to change the size relations between the child controls.
 	</description>
 	<tutorials>
 		<link title="Using Containers">$DOCS_URL/tutorials/ui/gui_containers.html</link>
 	</tutorials>
 	<methods>
 		<method name="clamp_split_offset">
 			<return type="void" />
+			<param index="0" name="priority_index" type="int" default="0" />
 			<description>
-				Clamps the [member split_offset] value to not go outside the currently possible minimal and maximum values.
+				Clamps the [member split_offsets] values to not go outside the possible minimum and maximum values, and to not overlap with each other. Use [param priority_index] to prioritize one split offset when there are overlaps.
```

Comment:
What does "prioritize" mean here? Is it more likely to get clamped or less, or what?

Reply:
It definitely needs to be better clarified, but not sure about the exact wording.

CC @Mickeon

Reply:
Updated the wording: "Clamps the [member split_offsets] values to ensure they are within valid ranges and do not overlap with each other. When overlaps occur, this method prioritizes one split offset (at index [param priority_index]) by clamping any overlapping split offsets to it."